javascript/ql/src/semmle/javascript/DefUse.qll

```diff
@@ -182,14 +182,26 @@ class VarDef extends ControlFlowNode {
   /** Gets a variable defined by this node, if any. */
   Variable getAVariable() { result = getTarget().(BindingPattern).getAVariable() }
 
+  private AST::ValueNode getTargetSource(Expr target) { defn(this, target, result) }
```

Comment:
I find the name of this predicate very confusing. Could we just inline it into its two uses?